src/jit/gentree.cpp

```diff
@@ -10694,7 +10694,8 @@ void Compiler::gtDispConst(GenTree* tree)
         case GT_CNS_INT:
             if (tree->IsIconHandle(GTF_ICON_STR_HDL))
             {
-                printf(" 0x%X \"%S\"", dspPtr(tree->gtIntCon.gtIconVal), eeGetCPString(tree->gtIntCon.gtIconVal));
+                const wchar_t* value = eeGetCPString(tree->gtIntCon.gtIconVal);
+                printf(" 0x%X \"%S\"", dspPtr(tree->gtIntCon.gtIconVal), value ? value : W("(null)"));
```

Comment:
Oh, I'm not sure about the semantics of ``printf`` in PAL, and thus tried not to affect other code instead of fixing ``printf``.

 I'll wait your fix.